code/modules/multiz/zmimic/mimic_movable.dm

```diff
@@ -9,7 +9,7 @@
 		if (isturf(dest))
 			bound_overlay.forceMove(get_step(src, UP))
 			if (dir != bound_overlay.dir)
-				bound_overlay.set_dir(dir)
+				bound_overlay.setDir(dir)
```

Comment:
yet again pizza proceeds to rename 1 proc for no reason and then change its name in 400 files